services/app-api/serverless.yml

```diff
@@ -464,15 +464,6 @@ functions:
           method: get
           cors: true
           authorizer: ${self:custom.authValue.${self:custom.stage}, ""}
-  uncertified:
-    handler: handlers/notification/uncertified.main
-    role: LambdaApiRole
-    events:
-      - http:
-          path: notification/uncertified
```

Comment:
This is called from the UI and needs to be removed there as well or it will 404.

Check out sendUncertifyEmail in api.js and its use in CertificationTab.js

Reply:
Just pushed up a change!